Doc/Makefile

```diff
@@ -49,7 +49,7 @@ PDFs := $(foreach dir, $(IMAGEDIRS), $(patsubst %.svg,%.pdf,$(wildcard $(SOURCED
 images: $(PDFs)
 
 clean:
-	-rm -rf $(BUILDDIR)/* source/autosummary/*.rst source/autosummary/autosummary/*.rst $(PDFs)
+	-rm -rf $(BUILDDIR)/* source/autosummary build $(PDFs)
```

Comment:
This is removing the entire build directory (and assuming its name). I'm okay with removing the entire autosummary directory instead of individual .rst, but let's not remove ``build`` (which may or may not be the build directory).

Reply:
This one was an easy fix, I just took out the "build"